libmscore/articulation.cpp

```diff
@@ -648,6 +681,7 @@ void Articulation::resetProperty(P_ID id)
       switch (id) {
             case P_ID::DIRECTION:
             case P_ID::TIME_STRETCH:
+            case P_ID::ORNAMENT_STYLE: // TODO not sure whether ornament style needs resetProperty handling.
```

Comment:
Hi Nicolas, not sure I understand this one.  Is it a problem that DIRECTION and TIME_STRETCH don’t implement a useful resetProperty?


> On 13 May 2015, at 11:38, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/articulation.cpp <https://github.com/musescore/MuseScore/pull/1946#discussion_r30215790>:
> 
> > @@ -648,6 +681,7 @@ void Articulation::resetProperty(P_ID id)
> >        switch (id) {
> >              case P_ID::DIRECTION:
> >              case P_ID::TIME_STRETCH:
> > +            case P_ID::ORNAMENT_STYLE: // TODO not sure whether ornament style needs resetProperty handling.
> About the TODO, I guess it does. It can be reset to default?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/1946/files#r30215790>.
>